### DISCUSSION TITLE
White Label

### DISCUSSION THREAD

**Discussion by Marys-Team** [chatwoot/chatwoot#7225] [Feature Requests]
Dear Chatwoot Team,

I hope this message finds you well. I am reaching out to discuss a feature request for Chatwoot that I believe would greatly benefit your users. I would like to suggest the addition of a white-label feature, allowing users to customize the appearance of the Chatwoot interface with their own logo and company name.

Currently, when using Chatwoot, the platform displays the Chatwoot logo and name. While this is perfectly fine for many users, there are instances where it would be more advantageous to have the ability to showcase their own branding. By offering a white-label option, users would be able to seamlessly integrate Chatwoot into their websites and applications, ensuring a consistent brand experience for their customers.

This feature would be particularly valuable for businesses that rely on maintaining a strong brand presence throughout their customer interactions. Being able to add their own logo and company name to the Chatwoot interface after moving it to their own domain would enhance trust, reinforce their brand identity, and create a cohesive customer experience.

Additionally, a white-label feature would be of great benefit to resellers or agencies who provide Chatwoot as part of their service offering. It would allow them to offer a fully branded and customized customer communication solution to their clients, elevating their value proposition and overall client satisfaction.

I understand that implementing a white-label feature may require development effort, but I believe it would be a worthwhile investment. By providing this option, you would attract more users and increase customer satisfaction, ultimately solidifying Chatwoot as the go-to customer communication platform for businesses of all sizes.

Thank you for considering this feature request. I truly believe that a white-label option would greatly enhance the Chatwoot experience and provide immense value to your users. I look forward to hearing your thoughts on this matter.

Best regards,
Marys-Team.

**Comment by sojan-official** [comment]
ref: https://www.chatwoot.com/docs/self-hosted/faq#how-can-i-customise-the-chatwoot-branding-
